calico_containers/pycalico/netns.py

```diff
@@ -61,201 +51,153 @@ def setup_logging(logfile, level=logging.INFO):
 
-    :param container_pid: The PID of the namespace to operate in.
-    :param ip: The IPAddress to remove.
-    :param interface_name: The interface to remove the address from.
-    :param proc_alias: The location of the /proc filesystem on the host.
-    :return: None. raises CalledProcessError on error.
-    """
-    with NamedNamespace(container_pid, proc=proc_alias) as ns:
-        ns.check_call("ip -%(version)s addr del %(addr)s/%(len)s "
-                      "dev %(device)s" %
-                        {"version": ip.version,
-                         "len": PREFIX_LEN[ip.version],
-                         "addr": ip,
-                         "device": interface_name},
-                      shell=True)
-
-
-def set_up_endpoint(ip, hostname, orchestrator_id, workload_id, cpid, next_hop_ips,
-                    veth_name=VETH_NAME,
-                    proc_alias=PROC_ALIAS,
-                    mac=None):
+
+def move_veth_into_ns(cpid, veth_name_ns_temp, veth_name_ns):
     """
-    Set up an endpoint (veth) in the network namespace identified by the PID.
+    Move the veth into the namespace.
 
-    :param ip: The IP address to assign to the endpoint (veth) as Netaddr
-    IPAddress.
-    :param hostname: The host that this endpoint's workload resides on.
-    :param orchestrator_id: The orchestrator_id that this endpoint was created on.
-    :param workload_id: The workload_id that this endpoint resides on.
     :param cpid: The PID of a process currently running in the namespace.
-    :param next_hop_ips: Dict of {version: IPAddress} for the next hops of the
-    default routes namespace, as opposed to the root namespace.  If so, this
-    method also moves the other end of the veth into the root namespace.
-    :param veth_name: The name of the interface inside the container namespace,
-    e.g. eth1
-    :param proc_alias: The location of the /proc filesystem on the host.
-    :param mac: The interface MAC to use.  Set to None to auto assign a MAC.
-    :return: An Endpoint describing the veth just created.
+    :param veth_name_ns_temp: The temporary interface name of the veth that will be
+    moved into the namespace.
+    :param veth_name_ns: The name of the interface in the namespace.
+    :return: None. Raises CalledProcessError on error.
     """
-    assert isinstance(ip, IPAddress)
-
-    # Generate a new endpoint ID.
-    ep_id = uuid.uuid1().hex
-
-    iface = generate_cali_interface_name(IF_PREFIX, ep_id)
-    iface_tmp = generate_cali_interface_name("tmp", ep_id)
-
-    # Provision the networking.  We create a temporary link from the proc
-    # alias to the /var/run/netns to provide a named namespace.  If we don't
-    # do this, when run from the calico-node container the PID of the
+    # Configure the container end.  To do this we create a temporary link from
+    # the proc alias to the /var/run/netns to provide a named namespace.  If we
```

Comment:
Oh poop I'd removed that entire comment in a previous pull request.  Don't think this comment should even be here.  I'm removing.